content/news/2023.pml

```diff
@@ -55,6 +55,34 @@ Older News: <%=reflist%>
 -->
 
 
+<!----------------------------------------------------------->
+<div class="news-item tag-link" id="summit-2023-keynotes">
+  <a href="#summit-2023-keynotes"><span class="tag-span"># </span></a> 19 June 2023: Keynotes for seL4 summit 2023 announced
+</div>
+<div class="row" style="margin-left:5px; margin-right:5px">
+  <p>
+    We are pleased to announce two keynotes for the seL4 summit 2023.
```

Comment:
Change as above, but shorter (no abstracts) and add links to summit page.

We are pleased to announce that the two keynotes for the [seL4 summit 2023](link to summit page) will be Gage from NCSC and Sam Leffler from Google! Gage will talk about ["Scoping assurance activities with seL4"](link to abstract in summit page) and Sam about ["CantripOS: An OS for Ambient ML Applications"](link to abstract in summit page). 

~
Then bios.

Also: in this news item, I'd change the pictures to be 2 logos: NCSC and Google. 

And I'd add links to the logos (going to NCSC and google)